app/commands/git/sync_track.rb

```diff
@@ -0,0 +1,103 @@
+module Git
+  class SyncTrack < Sync
+    include Mandate
+
+    def initialize(track)
+      super(track, track.synced_to_git_sha)
+      @track = track
+    end
+
+    def call
+      update_git_repo!
```

Comment:
I think we should rename this `update` to `fetch` all the way through things for clarity.